multi_host_migration/tests/migration_multi_host_auto_converge.py

```diff
@@ -0,0 +1,498 @@
+                        vm.wait_for_migration(self.migration_timeout)
+                    except virt_vm.VMMigrateTimeoutError:
+                        raise error.TestFail("After kill stessapptest, "
+                                             "migration failed with "
+                                             "auto-converge off")
+                else:
+                    raise error.TestFail("Migration failed with "
+                                         "auto-converge on")
+            finally:
+                if self.session is not None:
+                    self.session.close()
+                vm.destroy(gracefully=False)
+
+        @error.context_aware
+        def post_migration_capability_load_host(
+                self, vm, cancel_delay, mig_offline, dsthost,
+                vm_ports, not_wait_for_migration, fd, mig_data):
+
+            """
+            set auto-converge off/on during migration
+
+            :param vm: vm object
+            :param cancel_delay: If provided, specifies a time duration
+                   after which migration will be canceled.  Used for
+                   testing migrate_cancel.
+            :param mig_offline: If True, pause the source VM before migration
+            :param dsthost: Destination host
+            :param vm_ports: vm migration ports
+            :param not_wait_for_migration: If True migration start but not
+                   wait till the end of migration.
+            :param fd: File descriptor for migration
+            :param mig_data: Data for migration
+            """
+
+            try:
+                mig_thread = utils.InterruptedThread(self.get_sar_output)
+                mig_thread.start()
+                vm.wait_for_migration(self.migration_timeout)
+                logging.info("Migration completed with auto-converge on")
+            except virt_vm.VMMigrateTimeoutError:
+                if set_auto_converge == "no":
+                    logging.info("migration would never finish with "
+                                 "auto-converge off")
+                    try:
+                        if self.need_cleanup:
+                            self.clean_up(self.kill_bg_stress_cmd, vm)
+                        vm.wait_for_migration(self.migration_timeout)
+                    except virt_vm.VMMigrateTimeoutError:
+                        raise error.TestFail("After kill stessapptest, "
+                                             "migration failed with "
+                                             "auto-converge off")
+                else:
+                    raise error.TestFail("Migration failed with "
+                                         "auto-converge on")
+            finally:
+                if self.session is not None:
+                    self.session.close()
+                vm.destroy(gracefully=False)
+                mig_thread.join()
+
+        @error.context_aware
+        def post_migration_capability_load_host_io(
```

Comment:
use prefix "post_migration_capability" in super class and sub class, so do not suggest to modify.